docs/connector-development/connector-builder-ui/overview.md

```diff
@@ -1,57 +1,48 @@
+Generally, you can build a connector with the Connector Builder if you want to connect to an HTTP API that returns a collection of records as JSON and has fixed endpoints. For more detailed information on requirements, refer to the [compatibility guide](./connector-builder-compatibility.md). 
 
-When building a connector, it's important to differentiate between the connector, the configured source based on a connector, and the connection:
+## Getting started
 
-The **connector** defines the functionality of how to access an API or a database, for example protocol, URL paths to access, the way requests need to be structured, and how to extract records from responses.
+The step-by-step process for using the connector builder is as follows:
 
-:::info
-While configuring a connector in the builder, make sure to not hardcode things like API keys or passwords - these should be passed in as user input when configuring a Source based on your connector.
+1. Access the connector builder in the Airbyte web app by selecting "Builder" in the left-hand sidebar. 
+2. Iterate on your low-code connector by providing details for global configuration and user inputs. User inputs are the variables your connector will ask an end-user to provide when they configure a connector for use in a connection. 
+3. Once the connector is ready, publish it. This makes it available in your local workspace
+4. Configure a Source based on the released connector
+5. Use the Source in a connection to sync data
 
-Follow [the tutorial](./tutorial.mdx) for an example of how this looks like in practice.
-:::
+The concept pages in this section of the docs share more details related to the following topics: [authentication](./authentication.md), [record processing](./record-processing.mdx), [pagination](./pagination.md), [incremental sync](./incremental-sync.md), [partitioning](./partitioning.md), and [error handling](./error-handling.md). 
 
-The **configured source** is configuring a connector to actually extract records. The exact fields of the configuration depend on the connector, but in most cases, it provides authentication information (username and password, API key) and information about which data to extract, for example, the start date to sync records from, a search query records have to match.
+Do not hardcode things like API keys or passwords while configuring a connector in the builder. They will be used, but not saved, during development when you provide them as Testing Values. For use in production, these should be passed in as user inputs after publishing the connector to the workspace, when you configure a source using your connector. 
 
-The **connection** links up a configured source and a configured destination to perform syncs. It defines things like the replication frequency (e.g. hourly, daily, manually) and which streams to replicate.
+Follow [the tutorial](./tutorial.mdx) for an example of what this looks like in practice.
```

Comment:
Is the tutorial related to testing the connector, or is it more closely related to the `Getting started` steps? If it is the Getting Started part, I'd consider moving this link up higher, underneath the steps.

Reply:
This calls attention to the fact that the tutorial illustrates how Testing Values are used for the benefit of any reader who is uncertain about the role API keys and passwords play during development.